src/option_learning.py

```diff
@@ -212,3 +220,206 @@ def update_segment_from_option_spec(self, segment: Segment,
                                   dtype=np.float32)
                 option = param_opt.ground([robby, block], params)
                 segment.set_option(option)
+
+
+@dataclass(frozen=True, eq=False, repr=False)
+class _LearnedSimpleParameterizedOption(ParameterizedOption):
+    """A convenience class for holding a learned parameterized option.
+
+    Prefer to use this because it is pickleable.
+    """
+    _operator: STRIPSOperator
+    _regressor: MLPRegressor
+    _changing_parameter_idxs: Sequence[int]
+
+    def __init__(self, name: str, operator: STRIPSOperator,
```

Comment:
Why do you have both a dataclass and an `__init__`? I would say just have the `__init__` so you can avoid the hacks below.